package.json

```
@@ -0,0 +1,35 @@
+{
+  "name": "pantheon-geolocation-shortcodes",
+  "version": "0.2.0",
```

Should this match the version in [plugin.php](https://github.com/pantheon-systems/pantheon-geolocation-shortcodes/pull/3/files#diff-63e630aa779572c3d4c2bc7edf2dc6ae2da5e6719b1495ab2682fdc983bc2d96)?

```suggestion
  "version": "0.2.1",
```

---

Yep, probably!